### PR TITLE
Prerak Profile View

### PR DESCRIPTION
**Pull request by Dhanashree-Patil95**

### CODE REVIEW

File: apps/front-end/src/pages/front-end/FacilitatorQualification.js [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+    setfacilitator(result);
+  };
+  console.log("facilitator", facilitator);
+  return (
+    <Layout _appBar={{ name: t("BASIC_DETAILS") }}>
+      {facilitator?.experience.map((exp, index) => (
+        <VStack
+          paddingBottom="64px"
+          bg="bgGreyColor.200"
+          paddingLeft="16px"
```
Comment (Ekta29yadav) [inline apps/front-end/src/pages/front-end/FacilitatorQualification.js:47]: Can you refer ui structure from BeneficiaryEnrollment.js file

File: apps/front-end/src/pages/front-end/FacilitatorQualification.js [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+          paddingRight="16px"
+          space="24px"
+        >
+          <Box
+            marginTop="20px"
```
Comment (Ekta29yadav) [inline apps/front-end/src/pages/front-end/FacilitatorQualification.js:52]: Can you refer ui structure from BeneficiaryEnrollment.js file

File: apps/front-end/src/pages/front-end/FacilitatorExperience.js [hunk 3]
```diff
@@ -0,0 +1,296 @@
...
+                    color="textGreyColor.800"
+                    fontWeight="400"
+                    flex="0.4"
+                  >
+                    {facilitator?.alternative_mobile_number
+                      ? facilitator?.alternative_mobile_number
+                      : "-"}
+                  </FrontEndTypo.H3>
+                </HStack>
+                <Divider
```
Comment (Ekta29yadav) [inline apps/front-end/src/pages/front-end/FacilitatorExperience.js:260]: Can you refer ui structure from BeneficiaryEnrollment.js file check whole page

File: apps/front-end/src/pages/front-end/FacilitatorExperience.js [hunk 4]
```diff
@@ -0,0 +1,296 @@
...
+  return (
+    <Layout
+      _appBar={{
+        name:
+          type === "vo_experience"
+            ? t("VOLUNTEER_EXPERIENCE")
+            : t("WORK_EXPERIENCE"),
+      }}
+    >
+      <VStack
```
Comment (Ekta29yadav) [inline apps/front-end/src/pages/front-end/FacilitatorExperience.js:47]: Can you refer ui structure from BeneficiaryEnrollment.js file, check whole page

File: apps/front-end/src/pages/front-end/FacilitatorQualification.js (hunk 1 above)
Comment (Dhanashree-Patil95) [reply apps/front-end/src/pages/front-end/FacilitatorQualification.js]: resolved

File: apps/front-end/src/pages/front-end/FacilitatorQualification.js [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+
+                  <FrontEndTypo.H3
+                    color="textGreyColor.800"
+                    fontWeight="400"
+                    flex="0.4"
+                  >
+                    {facilitator?.mobile ? facilitator?.mobile : "-"}
+                  </FrontEndTypo.H3>
+                </HStack>
+                <Divider
```
[the remark this answers is not in the document]
Comment (Dhanashree-Patil95) [reply apps/front-end/src/pages/front-end/FacilitatorQualification.js:118]: resolved

File: apps/front-end/src/pages/front-end/FacilitatorQualification.js [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+                    color="textGreyColor.800"
+                    fontWeight="400"
+                    flex="0.4"
+                  >
+                    {facilitator?.alternative_mobile_number
+                      ? facilitator?.alternative_mobile_number
+                      : "-"}
+                  </FrontEndTypo.H3>
+                </HStack>
+                <Divider
```
[the remark this answers is not in the document]
Comment (Dhanashree-Patil95) [reply apps/front-end/src/pages/front-end/FacilitatorQualification.js:145]: Resolved

File: apps/front-end/src/pages/front-end/FacilitatorExperience.js (hunk 3 above)
Comment (Dhanashree-Patil95) [reply apps/front-end/src/pages/front-end/FacilitatorExperience.js]: Resolved

File: apps/front-end/src/pages/front-end/FacilitatorExperience.js (hunk 4 above)
Comment (Dhanashree-Patil95) [reply apps/front-end/src/pages/front-end/FacilitatorExperience.js]: Resolved

**Pull request merged**